tests/jballerina-unit-test/src/test/resources/test-src/bir/bir-dump/typedesc_bir

```diff
@@ -0,0 +1,227 @@
+================ Emitting Module ================
```

Comment:
Do we need this file

Reply:
Yes this is the assert file for `TypeDescBIRTest`